library_management_cookiecutter/library/views.py

```diff
@@ -0,0 +1,243 @@
+from django import http
+from django.shortcuts import render
+from django.shortcuts import redirect
+from django.views.generic import TemplateView, CreateView, ListView,FormView
+from .models import User, Book,AssignedBook
+from .forms import UserForm, AddBook,  AsignBook
+from django.contrib.auth.views import LoginView, LogoutView
+from django.views import View
+from django.http import HttpRequest, HttpResponse, JsonResponse
+from django.urls import reverse, reverse_lazy
+from django.contrib.auth import authenticate, login
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from .mixin import MyCustomPermissions
+from django.contrib.auth.models import Group, Permission
+from django.core import serializers
+from django.conf import settings
+from django.db.models import Count
+from django.utils import timezone
+import csv
+from django.core.mail import EmailMessage
+import zipfile
+import os
+import pyminizip
+from io import StringIO
+
+class Home(TemplateView):
+    template_name = "home.html"
+
+
+class Login(LoginView):
```

Comment:
why you need to add Login here. built in url available for Login,Logout and changepassowrd. ?